src/python/zquantum/qcbm/target_thermal_states.py

```diff
@@ -0,0 +1,235 @@
+import bisect
+import typing
+import numpy as np
+from zquantum.core.bitstring_distribution import BitstringDistribution
+from zquantum.core.utils import dec2bin, bin2dec, convert_tuples_to_bitstrings
+
+
+def int2ising(number: int, length: int) -> typing.List[int]:
```

Comment:
I would suggest renaming this to something more descriptive such as: `convert_integer_to_ising_bitstring`

Reply:
Also, do you think that this function would better fit in `z-quantum-core` or no? Seems like it could be used elsewhere beyond the QCBM

Reply:
Ah, I see the naming follows `dec2bin` and `bin2dec`. Pinging @max-radin and @mstechly for input on naming choice

Reply:
I don't think there was any special reason for using the names `dec2bin` and `bin2dec`. As I recall the names came from some other package. So something more explicit would probably be good.

Reply:
Done!